cloud-language-starter/src/main/java/com/sample/autoconfig/LanguageAutoConfig.java

```diff
@@ -96,7 +96,7 @@ public TransportChannelProvider defaultLanguageTransportChannelProvider() {
 
   @Bean
   @ConditionalOnMissingBean(name = "languageServiceSettings")
```

Comment:
```suggestion
  @ConditionalOnMissingBean
```
I think we also don't need a name here either. If user app has a ben of type `LanguageServiceSettings` then no need to setup this bean. Otherwise, this setting class is unique to the service already and we do not need to distinguish from other settings classes.